Cleanup installing docs (#19725)
matplotlib/matplotlib: INSTALL.rst

############
Installation
############

.. toctree::
   :hidden:

   installing_source.rst

==============================
Installing an official release
==============================

Matplotlib releases are available as wheel packages for macOS, Windows and
Linux on `PyPI <https://pypi.org/project/matplotlib/>`_. Install it using

> ``pip``::

Reviewer: this could also be tagged with `.. code-block:: sh`, or you could just use https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-highlight at the top instead.

   animations and a larger selection of file formats, you can
   install :ref:`optional_dependencies`.

=========================
Third-party distributions
=========================

Various third-parties provide Matplotlib for their environements.

Conda packages
==============

> Matplotlib is available both via the *anconda main channel*

Reviewer: anaconda (typo), also add a colon at the end (here and below)

   install :ref:`optional_dependencies`.

Although not required, we suggest also installing ``IPython`` for

> interactive use.

Reviewer 2: Suggest we delete this suggestion...

=========================
Third-party distributions
=========================

Various third-parties provide Matplotlib for their environements.

Conda packages
==============
Matplotlib is available both via the *anconda main channel*

.. code-block:: sh

   conda install matplotlib

as well as via the *conda-forge community channel*

.. code-block:: sh

   conda install -c conda-forge matplotlib

Python distributions
====================

Matplotlib is part of major Python distributions:

- `Anaconda <https://www.anaconda.com/>`_

- `ActiveState ActivePython
  <https://www.activestate.com/activepython/downloads>`_

- `WinPython <https://winpython.github.io/>`_

Linux package manager
=====================

> If you are using the Python version in your Linux distribution, you can install

Reviewer 2: ```suggestion
If you are using the Python that comes with your Linux distribution, you can install
```

> Matplotlib via your package manager:

Reviewer 2: ```suggestion
Matplotlib via your package manager.  e.g.:
```
